.travis.yml

```diff
@@ -1,235 +1,194 @@
-
-
+    - secure: h8n58Siclba6vRZEUcVzH2UTCBaA2src/mJvPaiXln1dPP4+/CICkfoljggA/qZDv9pv1ck07u7fHf6EJa7ONO2aNKD/r9CyyKHbZsgT5Wz6zLaOJfu/DmLZq9yclG/e9XmlK7qpjW+H7vCPn98GNVaU2h8cjqNVZ3KpkLwQ0j2RBoM6x4jpOFUsbiwnKWaHxS2u3JHpZ/L8PM9uy2JDoRm52HW5k+fmvyxwNis+R5ezteJJYcqhYhun9CRDVjGg47t8YFkO+fXjtgl5EAhNlOtE1UixV7h8+MVbmS0XJQm4Xen76J0a2O3xEGOggsdMJebTuDlHk97jhR17pMFu0lfM4V2YmiC8BclY/fdnxX7aoc9hPi9Zf6ReHOqVo8jWP4ArS7awHdKW8c5zkH2itf7M6tw89hnDmoXVk/48joOyj+vMwhvI2wJ3oUjhsjpZWG7Mks4hY2CYmkA5n56K/6+qgPq19qZr4MKqW1zr8066TNzqa2ZtdbtSzIyP0jcXLyNOxhwXejyBnHTyJ7ZVbiNxyGTvLQcOP1UYNn1LxmDTJWvDCguXbwLQ2oHzD032xVuNCS2XjZ1dDFtEtEfDniVnOGosEzJWK2kkRCaOzW5GT+VunfZnN9j4MNZVfI1iCW3EQ4CI7n/Ct89y4YrezwRoFjvMQvhb64pVLwMeOp4=
+  cache:
+    npm: true
+    directories:
+      - $HOME/.cache/pip
+      - doc/framework/node_modules
+  addons:
+    apt:
+      packages:
+        - python
+        - python-pip
+  install:
+    - pip install awscli --upgrade --user
+  script:
+    - npm run doc-prepare
+    - npm run doc-build
+  deploy:
+    provider: script
+    script:
+      - npm run doc-upload
+    skip_cleanup: true
+  after_deploy:
+    - npm run doc-cloudfront
+
+# ------------------------
+# Jobs configuration
+# ------------------------
 jobs:
   include:
-
-# # ---------------------------------------
-# # Unit Tests & Linters
-# # ---------------------------------------
-
+    # ---------------------------------------
+    # Unit Tests & Linters
+    # ---------------------------------------
     - stage: Unit Tests & Linters
-      if: type = pull_request OR (type = push AND branch =~ ^master|.*-dev|.*-stable$) OR type = cron
       name: Unit Tests & ESLint
       os: linux
       sudo: required
       language: node_js
-
-      cache: npm
-
-      node_js:
-        - "8"
-
+      node_js: 10
+      cache:
+        npm: true
+        directories:
+          - node_modules
       install:
-        - npm install --silent
-        - npm install --silent --only=dev
-
+        - npm ci --silent
       script:
         - npm run --silent lint
         - npm run unit-testing
-
       after_script:
         - npm run codecov
 
     - stage: Unit Tests & Linters
-      if: (type = pull_request AND fork != true) OR (type = push AND branch =~ ^master|.*-dev|.*-stable$) OR type = cron
       name: SonarQube
+      if: fork != true
```

Comment:
why?

Reply:
Because Sonarqube token is linked to kuzzleio Github account. So if community contributors want to open PR, this job will fail all the time with a `forbidden` error